romana-install/romana-setup

```diff
@@ -17,14 +17,15 @@
 
 usage() {
 	echo "romana-setup: executes the ansible playbooks for installing romana"
-	echo "Usage: romana-setup [-n name] [-p platform] [-d distro] [-s stacktype] [action]"
-	echo "       romana-setup [-n name] [-p platform] [-d distro] [-s stacktype] <action> [ansible-options]"
+	echo "Usage: romana-setup [-n name] [-p platform] [-d distro] [-s stacktype] [-c number] [action]"
+	echo "       romana-setup [-n name] [-p platform] [-d distro] [-s stacktype] [-c number] <action> [ansible-options]"
 	echo "       romana-setup <h|--help>"
 	echo ""
 	echo "Cluster Name:  user-defined stack name (default: $USER)"
 	echo "Platforms:     aws (default), vagrant, static"
 	echo "Distro:        ubuntu (default), centos"
 	echo "Stack Types:   devstack (default), kubernetes"
+	echo "Number:        number of compute nodes"
```

Comment:
But I mean that maybe it should be the option for the TOTAL cluster size. So, you'd update the code to ensure that it has to be between 1 and 199 (not 0 and 199) and simply add the comment that it's the total size and one host will also be used as controller. That way, it's cluster type independent and very clear about what will be created.